### ISSUE TITLE
MA0046 should not report when the delegate match the event signature

### ISSUE THREAD

**Issue by meziantou** [meziantou/Meziantou.Analyzer#60]
```csharp
public delegate void DataReceivedEventHandler(object sender, System.Diagnostics.DataReceivedEventArgs e);
public event System.Diagnostics.DataReceivedEventHandler ErrorDataReceived { add { } remove { } }
```
